server/db/models/user.js

```diff
@@ -1,7 +1,5 @@
 'use strict';
```

Comment:
General thought would be before you run the migration, would be to update this schema to mimic what's returned from G+ oauth (or whatever oauth we decide to use). eg add in profile image link etc